services/src/main/java/config/MainConfig.java

```diff
@@ -0,0 +1,16 @@
+package config;
+
+import api.TireFacade;
+import fascadeImpl.TireFacadeImpl;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import services.MappingService;
+
+/***
+ * @author Jakub Palenik, 422453@mail.muni.cz
+ */
+
+@Configuration
+@ComponentScan(basePackageClasses = {TireFacadeImpl.class})
```

Comment:
We should probably use a dedicated marker class, to stay consistent with [this](https://github.com/tinnyx/PA165-Pneuservis/blob/47743f6feb2724dd104ed6e024e49a9c556fd8af/backend/src/main/java/cz/muni/fi/pa165/pneuservis/backend/PersistenceApplicationContext.java#L28).